inventory/aws/ec2.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
I see. Ok, that's fine, but when we update this script and the gce.py script, whoever does it will likely not know to make this change, and so it will break you. If you can find another alternative, that would be best, otherwise I can see this as an ongoing issue for you. That being said, I won't block this PR on this.